assets/javascripts/discourse/templates/headlines/favorite-domains.hbs

```diff
@@ -35,4 +33,8 @@
       {{/each}}
     </table>
   {{/if}}
-</div>
+{{else}}
+  <div class="text--center">
+    Please <a href="#" class="color--primary" {{action "login"}}>log in</a> to build the list of your favorite sites
```

Comment:
Can we please change it to "Please log in to add your sites and receive scan reports over email"